server/lib/environment.js

```diff
@@ -1,3 +1,5 @@
+/*jshint newcap:false */
+
```

Comment:
@humphd Sure! Thank you. I just saw in  the previous examples with jshint directives a blank line was left and that is why I left it; however I will fix it. 
Just a small question. 
Here:
```js
/*jshint newcap:false */
```
Should I keep the extra space between false and * (same as it is used by jshint developers), or just get rid of the space? Thank you